libs/remote_storage/src/s3_bucket.rs

```diff
@@ -427,10 +427,12 @@ impl RemoteStorage for S3Bucket {
     }
 
     async fn download(&self, from: &RemotePath) -> Result<Download, DownloadError> {
+        // if prefix is not none then download file `prefix/from`
+        // if prefix is none then download file `from`
         self.download_object(GetObjectRequest {
             bucket: self.bucket_name.clone(),
             key: self.relative_path_to_s3_object(from),
-            ..GetObjectRequest::default()
+            range: None,
```

Comment:
Why do you need change, isn't `None` default?

Reply:
change not needed, but `range: None` is simpler IMO. I don't have a strong preference though